utils/service_discovery/sd_docker_backend.py

```diff
@@ -136,7 +140,7 @@ def _get_checks_to_refresh(self, state, c_id):
             return
 
         identifier = inspect.get('Config', {}).get('Labels', {}).get(DATADOG_ID) or \
-            inspect.get('Config', {}).get('Image')
+            DockerUtil().image_name_extractor(inspect)
```

Comment:
nitpick because it's not too expensive, but you should create a `self.dockerutil` in `__init__` and use `self.dockerutil` when you need it. That would avoid the existing instance lookup at every call.